modules/data_encoding/metadata_encoding_decoding.py

```diff
@@ -0,0 +1,75 @@
+"""
+Encoding and Decoding Metadata
+Save first byte as char to represent which worker sent the message 
+"""
+
+import struct
+
+from . import worker_enum
+
+
+# For Floats
+def encode_metadata(
+    worker_name: str, number_of_messages: int
+) -> "tuple[True, bytes] | tuple[False, None]":
+    """
+    Encode PositionGlobal object into Bytes. Worker_ID to be encoded as the first byte of the message
+
+    Parameters:
+       worker_name: name of the worker defined by the name of its own file (worker_name = pathlib.Path(__file__).stem)
+       number_of_messages: number of messages intended to be sent
+
+    Returns:
+        packed_coordinates (bytes): Encoded int corresponding to number of messages as bytes.
+        First byte dependant on which worker is calling the funciton, value depends on its corresponding enum value (see worker_enum.py)
+    """
+
+    try:
+
+        worker_id = worker_enum.WorkerEnum[worker_name.upper()]
+        if not worker_id:  # If worker ID is not in the Enum Class
+            return False, None
+
+        # Encode message using PositionGlobal's latitude, longitude, altitude, with the worker ID in the front
+        packed_metadata = struct.pack(
+            "=Bi",
+            worker_id.value,  # 1 byte (char)
+            number_of_messages,  # 4 bytes
+            # 5 bytes total
+        )
+    except (struct.error, AttributeError, ValueError):
+        return False, None
+
+    return True, packed_metadata
+
+
+def decode_metadata(
+    encoded_metadata: bytes,
+) -> "tuple[True, worker_enum.WorkerEnum, int] | tuple[False, None, None]":
+    """
+    Decode bytes into a PositionGlobal object.
+
+    Parameters:
+        encoded_message (bytes): Encoded bytearray containing Worker message ID, number of messages sent
+
+    Returns:
+        Tuple: success, WorkerEnum member instance corresponding to ID, number of messages received as an integer
+    """
+
+    # check to make sure encoded message is 1 char and 1 int
+
```

Comment:
Remove repeated comment + spaces (you explained on line 65 already, so it should be fine)